### ISSUE TITLE
Initialization models and accuracy@1  in pytorch

### ISSUE THREAD

**Issue by Lovelyczl** [mzolfaghari/ECO-efficient-video-understanding#49]
Thanks for your excellent work!
 I want to reproduce your paper，But I have some questions about Initialization models and accuracy@1 
Q1: does the models use Initialization models in your paper in ECO-pytorch if you just set args.pretrained_parts == "both", In other words,  weight_url_2d='https://yjxiong.blob.core.windows.net/ssn-models/bninception_rgb_kinetics_init-d4ee618d3399.pth' and models/C3DResNet18_rgb_16F_kinetics_v1.pth.tar
is your Initialization models in your paper  ECO-pytorch
Q2: is it normal that baseline  is 53.4% when i set args.pretrained_parts == "scratch" and num_segments=16 , could you provide some information about your model accuracy@1 when you set pretrained_parts='scratch', '2D', '3D', 'both','finetune' and segments=16
i am new learner in this, If you can help me, I will be very grateful.
Looking forward to your reply @mzolfaghari

**Comment by mzolfaghari** [comment]
@Lovelyczl 
Q1: yes this is the way we trained the models on Kinetics. But if you are using pre-trained kinetics models, then you can set args.pretrained_parts == "finetune".

Q2: The results depends on the dataset. But usually "finetune" is the best choice and "both" is the next choice. If you start from scratch usually the performance is low and takes longer to train the model. But as I said this depends on the dataset.

**Comment by Lovelyczl** [comment]
Thank you for your reply
i get it, it really help.
another naive question, hh
in fact, i trained the models in ucf101, but it has three splits. is it means that i should train and val my models in each spilt separately in total, and get the average value of that three results in the end ?
